api/api.go

```diff
@@ -637,11 +637,11 @@ func (api *api) SignMessage(ctx context.Context, message string) (*SignMessageRe
 	}, nil
 }
 
-func (api *api) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, sendAll bool) (*RedeemOnchainFundsResponse, error) {
+func (api *api) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, feeRate float64, sendAll bool) (*RedeemOnchainFundsResponse, error) {
```

Comment:
I don't think the feeRate needs to be a float, does it? it's in sats/vByte, how many users will choose e.g. `2.5`?

Reply:
But the highest fee (default setting) which comes from the mempool api can be a float right? Or is it set to be an integer?